### PR TITLE
Create release guide (#59)

### PR DESCRIPTION
**Pull request by dsotirho-ucsc**
https://github.com/databiosphere/hca-metadata-api/issues/59

### CODE REVIEW

**Review by achave11-ucsc — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -38,3 +38,39 @@ area repository.
 
 Copy the token and use it as the value of an environment variable named
 `GITHUB_TOKEN`.
+
+## Release Guide
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
## Release guide
```

Precedent.

File: README.md [hunk 2]
```diff
@@ -38,3 +38,39 @@ area repository.
...
+
+This guide will walk you through the steps necessary to promote the `develop`
+branch to `main`, tag the release, and prepare `develop` for the next release.
+
+Note: For purpose of example, this guide will pretend we are upgrading `main`
+from release 29 to 30.
+
+### Step 1: Merge `develop` Into `main`
```
Comment (hannes-ucsc) [inline README.md]: Nested numbered list would be less verbose and wouldn't clutter the TOC.

File: README.md [hunk 3]
```diff
@@ -38,3 +38,39 @@ area repository.
...
+- Resolve conflicts in setup.py, promote to the higher version.
```
Comment (hannes-ucsc) [inline README.md]: Empty lines between paragraphs. A heading is a special paragraph.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -38,3 +38,39 @@ area repository.
...
+## Release guide
+
+This guide will walk you through the steps necessary to promote the `develop`
+branch to `main`, tag the release, and prepare `develop` for the next release.
+
+Note: For purpose of example, this guide will pretend we are upgrading `main`
+from release 29 to 30.
+
+1. Merge `develop` into `main`.
+   - Resolve conflicts in setup.py, promote to the higher version.
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
   - Resolve conflicts in `setup.py`, promote to the higher version.
```

Comment (hannes-ucsc) [reply README.md]: Ideally, empty lines around 2nd level bullet items

File: README.md [hunk 5]
```diff
@@ -38,3 +38,39 @@ area repository.
...
+       ```
+       <     version="1.0b29",       # value from `main`
+       <     version="1.0b30.dev1",  # value from `develop`
+       >     version="1.0b30",       # new value for `main`
+       ```
+   - Title the merge commit: "Release beta 30".
+   - Push `main` to GitHub.
+
+2. Create a new release.
+   - GitHub -> Releases -> Draft a new release
```
Comment (hannes-ucsc) [inline README.md]:
```suggestion
   - On the GitHub epository home page click *Releases* — *Draft a new release*
```

Comment (hannes-ucsc) [reply README.md]: That's an mdash and italics for UI elements. Here and where applicable.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
The build isn't green. Probably because some interaction with #66 which I added as a blocker. Please hold this warm.

**Review by hannes-ucsc — APPROVED** [review]

**Pull request merged**